google/datalab/bigquery/commands/_bigquery.py

```diff
@@ -376,17 +376,45 @@ def _get_query_parameters(args, cell_body):
   if config:
     jsonschema.validate(config, BigQuerySchema.QUERY_PARAMS_SCHEMA)
 
+    # We merge the parameters with the airflow macros so that users can specify certain airflow
+    # macro names (like '@ds') in their sql. This is useful for enabling the user to progressively
+    # author a pipeline with %bq pipeline.
+
+    import datetime
+    canned_parameters = {
```

Comment:
You probably want to get the timestamp value once, and format all versions using that value.